### ISSUE TITLE
‘UINT_MAX’ undeclared in _libssh2_packet_add on 1.8.1

### ISSUE THREAD

**Issue by RazerM** [libssh2/libssh2#330]
```bash
$ cmake --build .
[  4%] Building C object src/CMakeFiles/libssh2.dir/packet.c.o
/.../libssh2-1.8.1/src/packet.c: In function ‘_libssh2_packet_add’:
/.../libssh2-1.8.1/src/packet.c:821: error: ‘UINT_MAX’ undeclared (first use in this function)
/.../libssh2-1.8.1/src/packet.c:821: error: (Each undeclared identifier is reported only once
/.../libssh2-1.8.1/src/packet.c:821: error: for each function it appears in.)
```

I assume `packet.c` doesn't include `libssh2.h`.

master branch compiles, I think 31d0b1a8530b959bd12c2074dc6e883e1eda8207 fixed it.

**Comment by willco007** [comment]
Please try the latest commit on the 1.8.1 branch.

**Comment by RazerM** [comment]
It compiles

**Comment by willco007** [comment]
Great, thanks. Fixed with #330.
